Subject: Adjust phpMyAdmin vendor configuration to match Debian needs
 - setup generates configuration in /var
 - documentation is in /usr/share/doc
 - config file consists of several included files, so we skip mtime check
From: Michal Čihař <nijel@debian.org>
Forwarded: not-needed
--- a/libraries/vendor_config.php
+++ b/libraries/vendor_config.php
@@ -14,31 +14,31 @@
  * Path to changelog file, can be gzip compressed. Useful when you want to
  * have documentation somewhere else, eg. /usr/share/doc.
  */
-define('CHANGELOG_FILE', './ChangeLog');
+define('CHANGELOG_FILE', '/usr/share/doc/phpmyadmin/changelog.gz');
 
 /**
  * Path to license file. Useful when you want to have documentation somewhere
  * else, eg. /usr/share/doc.
  */
-define('LICENSE_FILE', './LICENSE');
+define('LICENSE_FILE', '/usr/share/doc/phpmyadmin/copyright');
 
 /**
  * Path to config file generated using setup script.
  */
-define('SETUP_CONFIG_FILE', './config/config.inc.php');
+define('SETUP_CONFIG_FILE', '/var/lib/phpmyadmin/config.inc.php');
 
 /**
  * Whether setup requires writable directory where config
  * file will be generated.
  */
-define('SETUP_DIR_WRITABLE', true);
+define('SETUP_DIR_WRITABLE', false);
 
 /**
  * Directory where configuration files are stored.
  * It is not used directly in code, just a convenient
  * define used further in this file.
  */
-define('CONFIG_DIR', './');
+define('CONFIG_DIR', '/etc/phpmyadmin/');
 
 /**
  * Filename of a configuration file.
@@ -58,5 +58,5 @@
 /**
  * Default value for check for version upgrades.
  */
-define('VERSION_CHECK_DEFAULT', true);
+define('VERSION_CHECK_DEFAULT', false);
 ?>
